preset/k3s/preset.go

```diff
@@ -47,21 +49,89 @@ import (
 )
 
 const (
-	defaultPort     = 48443
-	defaultVersion  = "latest"
-	defaultHostname = "localhost"
+	// defaultAPIPort is the default port that the K3s HTTPS Kubernetes API gets
+	// served over.
+	defaultAPIPort = 48443
+	// defaultVersion is the default k3s version to run.
+	defaultVersion = "v1.19.3-k3s3"
```

Comment:
If you agree to drop 1.19 support and focus on the latest versions only, this also should be bumped, regardless backward compatibility.

In fact this version fails on my mac now even with the upstream rancher image (it failed before as well), but latest version 1.26.3 works correctly.